packages/@aws-cdk/aws-chatbot/README.md

```diff
@@ -27,17 +27,21 @@ This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aw
 
 ```ts
 import * as chatbot from '@aws-cdk/aws-chatbot';
+import * as lambda from '@aws-cdk/aws-lambda';
 
 const slackChannel = new chatbot.SlackChannelConfiguration(this, 'MySlackChannel', {
   slackChannelConfigurationName: 'YOUR_CHANNEL_NAME',
   slackWorkspaceId: 'YOUR_SLACK_WORKSPACE_ID',
   slackChannelId: 'YOUR_SLACK_CHANNEL_ID',
 });
 
-slackChannel.addLambdaInvokeCommandPermissions();
-slackChannel.addNotificationPermissions();
-slackChannel.addSupportCommandPermissions();
-slackChannel.addReadOnlyCommandPermissions();
+const invokeCommandStatement = new iam.PolicyStatement({
+  effect: iam.Effect.ALLOW,
+  resources: ['RESOURCE_ARNS_ALLOWED'],
+  actions: ['INVOKING_ACTION_ALLOWED'],
+});
+
+slackChannel.addToRolePolicy(invokeCommandStatement);
```

Comment:
the lambda grant invoke removes the need for this.

Reply:
@MrArnoldPalmer Updated. please review